src/reference/modules/administration/pages/server.adoc

```diff
@@ -211,6 +211,20 @@ Event Categories::
 * Policy Node
 * Archives
 
+== Health Check
+To ensure the health of RUDDER, some checks are executed every 5 minutes by default. To change the period
```

Comment:
```suggestion
To ensure the health of Rudder, some checks are executed every 5 minutes by default. To change the period
```

Reply:
Let's try to be locally consistent for now, we'll get back on the global level at some point